File: tarea6.c
#include<stdio.h>
#include<stdlib.h>

 int main(int argc, char *argv[]) {
 int a,b,c,d;
 int sumaa=0, sumab=0;
 system("clear");
 printf("Da los 2 coeficintes del 1er polinomio: "); 
 scanf("%d %d",&a, &b);
 printf("\nDa los 2 coeficintes del 2er polinomio: "); 
 scanf("%d %d",&c, &d);
 sumaa=a+c;
 sumab=b+d;
 printf("\nRespuesta= %dx + %d \n",sumaa, sumab);
 return 0;
}

File: tarea6.scala
object Polinomio {
 def main(args: Array[String]){
 println("Da coeficientes del 1er polinomio:")
 val a = readInt()
 val b = readInt()
 println(s"$a x + $b")

 println("\nDa coeficientes del 2do polinomio:")
 val y = readInt()
 val z = readInt()
 println(s"$y x + $z")

 val sumaa = a + y
 val sumab = b + z
 println(s"La suma de los polinomios es: $sumaa x + $sumab");
 }
}
